src/component/Timer/index.jsx

```diff
@@ -45,6 +44,25 @@ const Timer = () => {
     clearTimeout(tickTimeoutId.current);
   };
 
+  const handleResetClick = () => {
+    setIsRunning(false);
+    clearTimeout(tickTimeoutId.current);
```

Comment:
This code is the same as the `handleStopClick`. Can you move it out to a shared function and just invoke that function in both places?

Reply:
I was not thinking like this way. now I move it out and put in new function and I return call new function inside both.